close_test.go

```diff
@@ -0,0 +1,64 @@
+package i3
+
+import (
+	"context"
+	"os"
+	"os/exec"
+	"testing"
+	"time"
+)
+
+// TestCloseSubprocess runs in a process which has been started with
+// DISPLAY= pointing to an Xvfb instance with i3 -c testdata/i3.config running.
+func TestCloseSubprocess(t *testing.T) {
+	if os.Getenv("GO_WANT_XVFB") != "1" {
+		t.Skip("parent process")
+	}
+
+	t.Run("close", func(t *testing.T) {
```

Comment:
No need to do this in a subtest, right? Just remove the t.Run line here.

Reply:
Ah, right. Done.